waku/node/v2/waku_types.nim

```diff
@@ -204,14 +212,22 @@ proc generateRequestId*(rng: ref BrHmacDrbgContext): string =
   brHmacDrbgGenerate(rng[], bytes)
   toHex(bytes)
 
-proc computeIndex*(msg: WakuMessage): Index =
-  ## Takes a WakuMessage and returns its Index 
+proc id*(msg: WakuMessage): MDigest[256] =
```

Comment:
unclear what this is

Reply:
It's an id for the message, theres a comment explaining it.

Reply:
that's not what I mean, what I mean is why we are changing the signature

Reply:
Oh, because I only need to gather the ID of the message at one point and not the complete index. So I seperated the 2.